src/js/lib/search.js

```diff
@@ -418,7 +418,9 @@ export const getResourceUrl = result => {
     // Non-page results should have full URLs, convert to CDN if it's an S3 URL
     try {
       const originalUrl = new URL(result.url)
-      const useCDN = originalUrl.hostname.match(/s3\.amazonaws\.com/)
+      const useCDN =
+        originalUrl.hostname.match(/s3\.amazonaws\.com/) &&
+        process.env["CDN_PREFIX"]
       return useCDN ? `/coursemedia${originalUrl.pathname}` : result.url
```

Comment:
Is the prefix `/coursemedia` or `coursemedia` in production?

Reply:
The URL path should be `https://<domain>/coursemedia/....` so either the prefix can be set to `/coursemedia`, or it can be `coursemedia` and this line can insert the `/`:

```
return useCDN ? `/${cdnPrefix}${originalUrl.pathname}` : result.url
```